step-release-vis/src/app/services/environment_test.ts

```diff
@@ -13,7 +14,23 @@ describe('EnvironmentService', () => {
     service = TestBed.inject(EnvironmentService);
   });
 
-  it('should be created', () => {
-    expect(service).toBeTruthy();
+  // TODO(naoai): write getPolygons test
+  it('#getPercentages should return 3 candidates with ~ equal percentages', () => {
+    const input: CandidateInfo[] = [{name:  '1', job_count: 666}, {name: '2', job_count: 667},
+      {name: '3', job_count: 667}];
+    // @ts-ignore
+    const resultMap: Map<string, number> = service.getPercentages(input);
+
+    expect(resultMap.get('1')).toEqual(34);
```

Comment:
can we have fractional percentages pls? at some point we will have .5 % canaries, and we might want to show them on the same graph (especially for mobile)

Reply:
Done :)